assets/models/system/bert-base-cased/spec.yaml

```diff
@@ -10,7 +10,18 @@ properties:
   finetune-recommended-sku: Standard_NC24rs_v3
   finetuning-tasks: text-classification, token-classification, question-answering
   inference-min-sku-spec: 2|0|7|14
-  inference-recommended-sku: Standard_DS2_v2
+  inference-recommended-sku: Standard_DS2_v2, Standard_D2a_v4, Standard_D2as_v4, Standard_DS3_v2,
```

Comment:
Why not add inference sku spec instead? 
This is a rather long list. Same at other such places.